games/battlecity/src/main/java/com/codenjoy/dojo/battlecity/model/levels/Level.java

```diff
@@ -99,6 +99,11 @@ public int size() {
         return result;
     }
 
+    @Override
+    public boolean isAmmoBonus(int x, int y) {
+        return false; // do nothing
```

Comment:
1) Ammo bonus should be removed after it's picked up by a tank
2) Ammo bonus will appear after configurable ticks (or randomly)
3) Tank shall be able to destroy Ammo bonus by Bullet